aiohttp/tracing.py

```diff
@@ -9,13 +9,14 @@
 
 
 __all__ = (
-    'TraceConfig', 'TraceRequestStartParams', 'TraceRequestEndParams',
-    'TraceRequestExceptionParams', 'TraceConnectionQueuedStartParams',
-    'TraceConnectionQueuedEndParams', 'TraceConnectionCreateStartParams',
-    'TraceConnectionCreateEndParams', 'TraceConnectionReuseconnParams',
-    'TraceDnsResolveHostStartParams', 'TraceDnsResolveHostEndParams',
-    'TraceDnsCacheHitParams', 'TraceDnsCacheMissParams',
-    'TraceRequestRedirectParams'
+    'TraceConfig', 'TraceRequestStartParams',
```

Comment:
Do you really need to touch all these lines?
Appending two classes to the end of sequence should be enough.

Reply:
Ok, reverted and added new classes at the end